configs/test/bigquery/datasets.yaml

```diff
@@ -225,3 +225,29 @@ resources:
             type: FLOAT
           - name: engine
             type: STRING
+  - name: cross-pollination-stats-table
+    type: gcp-types/bigquery-v2:tables
+    properties:
+      datasetId: $(ref.main-dataset.datasetReference.datasetId)
+      tableReference:
+        tableId: cross-pollination-statistics
+      timePartitioning:
+        type: DAY
+      schema:
+        fields:
+        - name: project_qualified_name
+          type: STRING
+        - name: tagged
```

Comment:
I guess the purpose here will eventually be to differentiate between the new implementation using explicit tags and normal cross pollination? Could this be something like a "method" string instead of a boolean to make it more resilient to whatever we might do in the future?

Reply:
This may not matter if you decide to include the list of tags. Up to you how you handle this, but either way I'd like to move away from the boolean.

Reply:
Done.